install_config/aggregate_logging.adoc

```diff
@@ -1207,6 +1207,15 @@ Setting the `loggingPublicURL` parameter creates a *View Archive* button on the
 {product-title} web console under the *Browse* -> *Pods* -> *<pod_name>* ->
 *Logs* tab. This links to the Kibana console.
 
+[NOTE]
+----
+You need to log in to the Kibana console when your valid login cookie expires after 168 hours (1 week), for example:
```

Comment:
168 hours?  The OCP valid  session will expire in 24 hours.  I guess the Kibana Login cookie should be same. Let me have a try.

Reply:
You need to log in to the kibana console when your valid OCP token expired.  the default is 24 hours.

Reply:
@josefkarasek Can you respond on Anping's comment on 168 hours? In an email, you suggested "the log in expiration period is 1 week (168hours) "

Reply:
@anpingli good point

@jcantrill we don't overwrite that, maybe we should openshift/openshift-ansible/pull/10160